lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -45,15 +42,11 @@ def test_bad_tar(self, caplog, valid_config):
         )
         expected_error_message = f"tarball does not exist, '{bad_tarball}'"
         caplog.set_level(logging.ERROR, logger=self.logger.name)
-        try:
+        with pytest.raises(FileNotFoundError) as e:
             crt = CopyResultTb("controller", bad_tarball, self.config, self.logger)
-            crt.copy_result_tb()
-        except SystemExit:
-            assert caplog.records
-            assert len(caplog.records) == 1
-            assert caplog.records[0].message == expected_error_message
-        else:
-            assert False
+            crt.copy_result_tb("token")
+        assert str(e.FileNotFoundError).endswith(expected_error_message)
```

Comment:
What's with the `e.FileNotFoundError`? `e` should _be_ a `FileNotFoundError` object. Also, didn't you capitalize the "Tarball does not exist" message?

Reply:
I was just trying to experiment things sorry about that. forgot to revert the changes. 
Yes it worked.. Yee.. Thank you :D. Forgot to capitalize